src/main/java/jobmanager/messaging/JobMessager.java

```diff
@@ -68,6 +68,13 @@
 	private UpdateStatusHandler updateStatusHandler;
 	private RepeatJobHandler repeatJobHandler;
 
+	@Value("${pz.workflow.event.id}")
```

Comment:
Since we have multiple events now, let's be more specific. Try `pz.workflow.event.abort.id`